pkg/broker/binding_operations.go

```diff
@@ -155,6 +155,8 @@ func (b Broker) Unbind(ctx context.Context, instanceID string, bindingID string,
 
 	client, p, err := b.getClient(ctx, instanceID, details.PlanID, nil)
```

Comment:
Please also prefix the name of the PR and the commit that lands with the CLOUDP reference 

ie. "CLOUDP-123123: Add conversion for old plans" 

This is to help us go back and look things up in the future 

A little more detail in the commit title might help - "CLOUDP-12312: Support old Plan data formats in Realm"

Reply:
Changed this. Will change the commit message during squashing.